tests/ui/argument-suggestions/basic.stderr

```diff
@@ -18,7 +18,7 @@ error[E0061]: this function takes 0 arguments but 1 argument was supplied
 LL |     extra("");
    |     ^^^^^ --
    |           |
-   |           unexpected argument of type `&'static str`
+   |           unexpected 1st argument of type `&'static str`
```

Comment:
Thank you for this PR. This seems very useful.

However, it's a bit odd to see `1st argument` when that's the only argument in the call.

Reply:
Exactly. I reverted the message as before if number of input or expected arguments is one.